### PR TITLE
Volunteer registration form with some tests

### PR DESCRIPTION
**Pull request by vzickner**
This is a partial implementation of #20:
After the OAuth 2.0 registration, it is necessary to save some user data. This pull request includes a form for user data and save them to the REST API. It is tested with the new protractor test behavior (see pull request #39 ).

Currently it is still work in progress:
- Detailed error handling is missing (highlight wrong field, display error message)
- Forward to 'thank you' page / please confirm page after successful creation

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1,13 +1,5 @@
 angular.module('Come2HelpController', ['ngResource']);
```
Comment (jGleitz) [inline js/controller.js:1]: I think we shouldn’t call our module “`Controller`”

**Comment by jGleitz** [comment]
Mocha tests have a fluent API. If you’d change the test descriptions, e.g. to

``` JavaScript
describe('Register Page', function() {
    it('has two entries: "Mocked Ability 1" and "Mocked Ability 2"', function() {
    });

    it('allows to register with correct data', function() {
    });

    it('shows an error for adult=false', function() {
    });
}); 
```

It’s very easy to read and we get a nice log, like:

```
Register Page
    ✓ has two entries: "Mocked Ability 1" and "Mocked Ability 2"
    ✓ allows to register with correct data
    ✓ shows an error for adult=false
```

File: test/behaviour/mocks/locations.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+			id: 2,
+			name: 'Transport',
+			latitude: 48.1232,
+			longitude: 8,
+			location: 'Rheinbad',
+			options: {
+				title: 'Transport im Rheinbad'
+			}
+		}, {
+			id: 2,
```
Comment (jGleitz) [inline test/behaviour/mocks/locations.js]: Duplicate id `2`!

File: test/behaviour/protractor.browserstack.conf.js [hunk 3]
```diff
@@ -100,7 +100,11 @@ exports.config = {
 		reporter: 'min'
 	},
 	specs: ['**/*.js'],
-	exclude: ['**/*.conf.js']
+	exclude: ['**/*.conf.js'],
```
Comment (jGleitz) [inline test/behaviour/protractor.browserstack.conf.js:103]: Is this still needed, as we have `http-mock.conf.js`?

File: test/behaviour/protractor.conf.js [hunk 4]
```diff
@@ -9,5 +9,9 @@ exports.config = {
 		slow: 3000
 	},
 	specs: ['**/*.js'],
-	exclude: ['**/*.conf.js']
+	exclude: ['**/*.conf.js'],
+	mocks: {
+		dir: 'mocks',
+		default: ['default']
+	}
```
Comment (jGleitz) [inline test/behaviour/protractor.conf.js:16]: Is this still needed, as we have `http-mock.conf.js`?

Comment (vzickner) [inline test/behaviour/protractor.conf.js:16]: This was already removed in 2cb90f1b81b0 (but is currently not merged to this branch).

File: test/behaviour/protractor.browserstack.conf.js (hunk 3 above)
Comment (vzickner) [inline test/behaviour/protractor.browserstack.conf.js:103]: This was already removed in 2cb90f1b81b0 (but is currently not merged to this branch).

File: js/controller.js (hunk 1 above)
Comment (vzickner) [inline js/controller.js:1]: Yes, we should improve the name of this module, but I think is independent of this pull request.

File: test/behaviour/mocks/locations.js (hunk 2 above)
Comment (vzickner) [inline test/behaviour/mocks/locations.js]: This mock file is not necessary. It was removed in e5b1be6284d40c5426e5b48b3945513aca9da984 .

**Comment by vzickner** [comment]
Any further comments on this pull request?

**Pull request merged**